### ISSUE TITLE
Retrieved annual cash flow of TCEHY is inconsistent with website

### ISSUE THREAD

**Issue by xwu64** [ranaroussi/yfinance#913]
Website:

![website](https://i.imgur.com/G8O8fNb.png)

Retrieved:
![retrieved](https://i.imgur.com/uPS4FHH.png)

For example, the operating cash flow in **12/31/2020** is **1.94119e8** instead of **1.94119e11**. 

Anyone meet the same issue?

**Comment by eabase** [comment]
@xwu64 
Can you provide a code snippet to reproduce the erroneous results?

**Comment by xwu64** [comment]
Sorry for missing code. I'm using example code shown in the README file. 

```python
import yfinance as yf

tcehy = yf.Ticker("TCEHY")
print(tcehy.cashflow)
```

**Comment by eabase** [comment]
@asafravid 
I wonder if this is a similar problem as in #903. Perhaps we're searching the wrong path here again?

**Comment by eabase** [comment]
@xwu64 
Can you check if the correct one is present in this table, when using `t.quarterly_cashflow`, instead?

**Comment by asafravid** [comment]
> @xwu64
> Can you check if the correct one is present in this table, when using `t.quarterly_cashflow`, instead?

@eabase definitely looks similar

**Comment by xwu64** [comment]
> @xwu64 Can you check if the correct one is present in this table, when using `t.quarterly_cashflow`, instead?

same issue

**Comment by xwu64** [comment]
I got the reason for difference in the operating cash flow. It's because the website should all numbers in thousands. I'm sorry for this silly mistake. 

However, the **Capital Expenditure** is still inconsistent.

**Comment by git-shogg** [comment]
@xwu64 The issue with the capital expenditure is resolved in my PR https://github.com/ranaroussi/yfinance/pull/776. You can use this if you like. Note that I have changed some of the function names to align with the statement names. You'll need to run the following two lines to get the right capital expenditure for TCEHY.
`stock_detail = yf.Ticker("TCEHY")
stock_detail.cash_flow_statement`
Hope this helps resolve your issue.

**Comment by ValueRaider** [comment]
Correct financials are now available in pre-release version 0.2.0rc4.
